src/components/notifications/GenericErrorToast.tsx

```diff
@@ -15,24 +15,41 @@
  ***************************************************************************** */
 
 import React from 'react';
+import { copyTextToClipboard } from '../../helpers/copyHandler';
+import { createStyleSelector } from '../../helpers/styleCreators';
 import { GenericError } from '../../stores/NotificationsStore';
 
 export default function GenericErrorToast(props: GenericError) {
 	const { description, header, action } = props;
+	const [copied, setCopied] = React.useState(false);
+	const copyDetailsText = createStyleSelector('toast-action__text', copied ? 'copied' : null);
+
+	const copy = () => {
+		copyTextToClipboard(description);
```

Comment:
Tried, didn't succeed.
I think then we'd have to overload callback, because currently it doesn't accept any arguments. but overloads have to work universally, and copyTextToClipboard doesn't accept empty arg.

I also tried extracting all copying logic to NotificationStore, but it was clumsy and didn't work too. Idk, if there are any additonal tips, i could try again